### PR TITLE
Cleanup for the basic replication model

### PR DESCRIPTION
**Pull request by Yekai-Yu**
保留一个最基本的model
需要augmentation的时候，新建branch来跑

### CODE REVIEW

File: replication_chexnet_cuda_local.py [hunk 1]
```diff
@@ -212,16 +209,17 @@ def train_model(model, train_loader, val_loader, n_epochs, logfile):
 
             gc.collect()
             torch.cuda.empty_cache()
+
             model.eval()
             log.write('AUROCs on validation dataset:\n')
-            val_loss = 0           
+            test_loss = 0
```
Comment (Yekai-Yu) [inline replication_chexnet_cuda_local.py]: rename

File: replication_chexnet_cuda_local.py [hunk 2]
```diff
@@ -238,8 +236,6 @@ def eval_model(model, test_loader, logfile):
     y_test = y_test.cuda()
     y_pred = torch.FloatTensor()
     y_pred = y_pred.cuda()
-    criterion = nn.BCELoss()
```
Comment (Yekai-Yu) [inline replication_chexnet_cuda_local.py:241]: removed dup

File: replication_chexnet_cuda_local.py (hunk 1 above)
Comment (lytinahome) [reply replication_chexnet_cuda_local.py]: I feel that lines 215, 217, 222 should be named as val_loss, since this is the loss on the validation dataset. I think using val loss as the variable name is more informative.

File: replication_chexnet_cuda_local.py (hunk 2 above)
Comment (lipingxie) [reply replication_chexnet_cuda_local.py:241]: criterion = nn.BCELoss() here is not duplicated, since it is within eval_model, we need this to calculate the validation loss

File: replication_chexnet_cuda_local.py (hunk 1 above)
Comment (lipingxie) [reply replication_chexnet_cuda_local.py]: Please don't rename it, val_loss is for calculating the validation loss from the validation test, we need this value in the scheduler step, here is the related code:
                val_loss = eval_model(model, val_loader, logfile)

            log = open(logfile, "a")
            log.write('Epoch: {} \tLearning Rate for first group: {:.10f}\n'.format(epoch+1, optimizer.param_groups[0]['lr']))
            model.train()
            scheduler.step(val_loss)

File: replication_chexnet_cuda_local.py (hunk 2 above)
Comment (Yekai-Yu) [reply replication_chexnet_cuda_local.py:241]: We already have it declared https://github.com/lulucikyo/CheXNet/pull/7/files#diff-6381ec9930c25103136d6d6047cd35e00bf3f056faad8af53c955b1cae1b1425L235

Comment (lytinahome) [reply replication_chexnet_cuda_local.py:241]: agreed, this should be removed as line 235 already defined the criterion.

File: replication_chexnet_cuda_local.py (hunk 1 above)
Comment (Yekai-Yu) [reply replication_chexnet_cuda_local.py]: Done

**Review by lytinahome — APPROVED** [review]

**Review by lipingxie — APPROVED** [review]
